src/main/scala/com/mozilla/telemetry/streaming/EventsToAmplitude.scala

```diff
@@ -202,7 +202,7 @@ object EventsToAmplitude {
 
   def sendStreamingEvents(spark: SparkSession, opts: Opts): Unit = {
     val config = readConfigFile(opts.configFilePath())
-    val apiKey = sys.env(AMPLITUDE_API_KEY_KEY)
+    val apiKey = sys.env.getOrElse(AMPLITUDE_API_KEY_KEY, "test-api-key")
```

Comment:
This changes production behavior: if the key is missing from the env, it will no longer error out. I'm open to removing the setting of the API key in `runtests.sh` if it can be added only during testing. If you can figure out an alternative for setting it, then we can remove `runtests.sh` altogether.

Reply:
I see. Assuming we don't want to change the way we provide key, how about:
* introducing new method accepting `Opts` and api key to act as entrypoint for processing
* calling this method from `main` with parsed `Opts` and key extracted from environment variable
* calling this method in test (instead of main with list of arguments)
* likewise, refactor `ErrorAggregator`

Reply:
@akkomar that sounds like a reasonable change to me, and should keep the test api key string consistent within the testing module.